ci/copy-unix-release-artifacts.sh

```diff
@@ -49,4 +49,13 @@ if [[ "$NAME" == "linux" ]]; then
     NON_REPUDIATION=non-repudiation-$RELEASE_TAG.jar
     bazel build //runtime-components/non-repudiation-app:non-repudiation-app_deploy.jar
     cp bazel-bin/runtime-components/non-repudiation-app/non-repudiation-app_deploy.jar $OUTPUT_DIR/github/$NON_REPUDIATION
+
+    NON_REPUDIATION_CLIENT_JAR=non-repudiation-client-$RELEASE_TAG.jar
+    NON_REPUDIATION_CLIENT_POM=non-repudiation-client-$RELEASE_TAG.pom
+    NON_REPUDIATION_CLIENT_SRC=non-repudiation-client-$RELEASE_TAG-sources.jar
```

Comment:
Should we include javadoc as well?

Reply:
Is there a specific target to run to get it? I didn't see it in the `bazel-bin` directory for `//runtime-components/non-repudiation-client`.

Reply:
This is the full extend of the javadoc in the library, though:

```
/**
 * A gRPC client-side interceptor that uses a key pair to sign a payload and adds it as metadata to
 * the call, alongside a fingerprint of the public key and the algorithm used to sign.
 */
 ```
 
 Maybe at this point it wouldn't make too much sense to publish it, unless it's important to have it for some reason I'm missing.

Reply:
I know Maven mandates having it but I think artifactory doesn’t. I’d probably still publish it. It seems easier to publish javadocs everywhere than to to a per-library decision. As for why the target is not there, it looks like we need to add the following tag for this to work:
```
        "javadoc_root_packages=com.daml.nonrepudiation.client",
```

Reply:
Mmmh, I remember trying to add it and having issues with that, I'll see if that's still the case, thanks for the tip.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/9282/commits/da3cc126e64751aa86902af5bc0b2833bf1c626c